docs/make_docs.py

```diff
@@ -66,10 +67,10 @@ def __init__(self, output_dir="python_api", input_dir="python_api_in"):
         self.output_dir = output_dir
         self.input_dir = input_dir
         self.module_names = PyAPIDocsBuilder._get_documented_module_names()
-        print("Generating *.rst Python API docs in directory: %s" %
-              self.output_dir)
 
     def generate_rst(self):
+        print("Generating *.rst Python API docs in directory: %s" %
```

Comment:
Could use `f"Generating *.rst Python API docs in directory: {self.output_dir}"` instead. I know the current doc is not consistent, but maybe it's better to gradually fix them.